src/lib/comms/sol-comms.c

```diff
@@ -74,8 +90,14 @@ sol_comms_init(void)
     if (ret != 0)
         goto http_server_error;
 
+    ret = sol_netctl_init();
+    if (ret != 0)
+        goto netctl_error;
```

Comment:
After checking it, yes. It is not necessary.
I will remove it in the next version.